.github/workflows/continuous-integration.yaml

```
@@ -21,7 +21,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        node-version: [10, 12, 14, 16]
+        node-version: [12, 14, 16]
```

```suggestion
        node-version: [10, 12, 14, 16]
```

See: https://github.com/loopbackio/strong-soap/pull/342#discussion_r754932014

---

@achrinza, i've updated this PR only containing the change in CODEOWNERS file. PTAL. Thanks.